src/ducks/balance/helpers.js

```diff
@@ -77,3 +78,21 @@ const sumAllBalancesByDate = accountsBalances => {
 
   return balances
 }
+
```

Comment:
Debate : I see the virtual groups more something that would belong in `ducks/accounts` than in balance.

Reply:
I think we can go further and create a new `groups` duck, no?

Reply:
Yes maybe :) As you like :)